tests/math.py

```diff
@@ -10,24 +10,24 @@
 from snake_egg import EGraph, Rewrite, Var, vars
 
 import unittest
-
+from typing import List, Any
 from collections import namedtuple
 
 
 # Operations
-diff = namedtuple("Diff", "x y")
-inte = namedtuple("Integral", "x y")
+diff = namedtuple("Diff", "x y") # type: ignore
```

Comment:
I reverted the rename of the named tuple types, instead opting to just ignore the error mypy raises that the variable name should match the name, to minimize the diff.

Reply:
Hmm. Can we either just capitalize this or ignore on a global level?

Reply:
👍 Sure I can refactor to capitalize them all